src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/VSSettings.cs

```diff
@@ -108,6 +108,7 @@ private void OnSolutionOpenedOrClosed(object sender, EventArgs e)
 
             if (hasChanged)
             {
+                var invokers = SettingsChanged.GetInvocationList();
```

Comment:
What was your plan with this unused local?

Reply:
oops, that was just for debugging, to make sure when I closed VS that my other changes worked as expected.